go/frontendbridge/main.go

```diff
@@ -156,21 +160,24 @@ func (w *FrontendBridge) Close() error {
 	if err != nil {
 		return err
 	}
-	w.connections = nil
+	w.openMutex.Lock()
 	w.open = false
+	w.openMutex.Unlock()
 	return nil
 }
 
 func (w *FrontendBridge) websocketHandler(socket *websocket.Conn) {
 	auth := map[string]interface{}{"sessionId": uuid.NewV4().String()}
 	w.addConnection(auth, socket)
 	utils.ListenForMessages(socket, func(message structs.Message) error {
+		w.exoRelayMutex.Lock()
 		_, err := w.exoRelay.Send(exorelay.MessageOptions{
 			Name:       message.Name,
 			Payload:    message.Payload,
 			ActivityID: message.ActivityID,
 			Auth:       auth,
 		})
+		w.exoRelayMutex.Unlock()
```

Comment:
the race detector alerted about a race condition in writing to the websocket of exorelay.